osi_object.proto

```diff
@@ -436,6 +437,86 @@ message MovingObject
         // through.
         //
         optional double ground_clearance = 6;
+
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 7;
+
+        // \brief Detailed wheel data.
+        // The focus is on the description of a wheel regarding the perceivable 
+        // information from the outside. 
+        // It is not intended to be used for e.g. dynamic calculations.
+        //
+        message WheelData
+        {
+            // The axle which contains this wheel. A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            // This concept works also for twin tires.
+            //
+            optional uint32 index = 2;
+            
+            // Position of the wheel (geomatric center of the wheel).
+            // The reference system is the vehicle frame (center of bounding box).
+            // Right-handed, x-axis shows into driving direction "forward".
+            // Rough Approximation: The z-value shows the damping.
+            //
+            optional Vector3d position = 3;
+
+            // Median radius of the wheel measured from the center of the wheel to
+            // the outer part of the tire.
+            //
+            //
+            // Unit: m
+            //
+            optional double wheel_radius = 4;
+
+            // Median radius of the rim measured from the center to the outer, visible part of the rim.
+            //
+            // Unit: m
+            //
+            optional double rim_radius = 5;
+            
+            // Median width of the tire.
+            //
+            // Unit: m
+            //
+            optional double width = 6;
+
+            // Orientation of the wheel.
+            //
+            // The reference system is the vehicle frame (center of bounding box).
+            // Right-handed, x-axis shows into driving direction "forward".
+            // Rough Approximation: The yaw-value shows the steering angle.
+            //
+            optional Orientation3d orientation = 7;
+
+            // Rotation rate in the wheel's frame of reference (rotation around the wheel axle).
+            //
+            // Unit: rad/s.
+            //
+            // The sign convention is defined using the right-hand rule with respect to the vehicle's coordinate system (DIN ISO 8855:2013-11):
```

Comment:
This might be a very noddy question, so sorry in advance!

This is defined as (counter-)clockwise rotation around the wheel axle, is that always when considered from the y-positive direction of the vehicle's co-ordinate system?  i.e. both wheels on an axle will typically have the same sign on their rotation rate.

Might be worth adding in a basic example to clarify.  (or let me know if this is so commonly understood by people who know about these things it isn't worth clarifying!)

Reply:
@caspar-ai , yes that is true, this means that both wheels on an axle will typically have the same sign on their rotation rate.
We also discussed adding pictures and examples, it was left for @ThomasNaderBMW  to do.

Reply:
Yes the "forward" concept was in my first proposal. But the definition is no very scientific, so we changed to the right hand rule.. But to be honest I still think that it is even more complicated now and not self-explaining..
@kmeids , what do you think. Shall we change to an easier convention?

Reply:
@ThomasNaderBMW, no need to change it, I think with a suitable pictogram there would be no problem understanding it.

Reply:
@kmeids ah ok. Could that be a task for the service provider? That would help a lot, especially to demonstrate it in an understandable way!

Reply:
@ThomasNaderBMW, yes it can be however in this case someone should still clarify the issue to the service provider.
Just added the "Documentation" label to it.

Reply:
Hi everyone, using an image to introduce and explain complex issues is a good idea from my point of view. Shall that image already be included in the next minor release?

Reply:
I think it would be great, because there was a lot of discussion already going on in the CCB meeting!

Reply:
@ThomasNaderBMW Can you provide an image that I can include in the documentation?

Reply:
CCB Meeting 25.11.2020:

Actions:
@ThomasNaderBMW to update the coordinate system reference point.
@paagkame to upload a pictogram illustrating the sign convention for rotation.

Reply:
I reworked the open actions with the last commit. I hope it is clearer now! :)

Reply:
I have created a new svg. Please have a look at it at open-simulation-interface\doc\images\OSI_RotationRate.svg
I am not sure though if the use of color is allowed and where to place the image best. The .proto file does not seem to be the right place.

Reply:
@paagkame 
I assume that by "F" you are referring to the moving direction (Forward).
This can be confused by F as in Force.
To evade any confusions, we should either state that F is referring to forward or just extend the arrow a bit linger and write "moving direction"

![image](https://user-images.githubusercontent.com/25367407/101239416-aa73e780-36e7-11eb-9399-f81fd51fe98d.png)

Reply:
@paagkame 
Could you also remove the "V" after the "+" and "-" sign?

Reply:
SVG is updated with proposed changes.